### DISCUSSION TITLE
Nominatim on RockyLinux

### DISCUSSION THREAD

**Discussion by netluca1** [osm-search/Nominatim#2997] [General]
Hello everyone.
I noticed that from version 4.1 the Centos 8 operating system or similar RockyLinux type is no longer offered.
In the new versions we only talk about ubuntu.
Is there a particular reason?
Are there any problems with this system?
I've always used centos and RockyLinux (excellent successor) and I've always found it good

**Comment by lonvia** [comment]
I had to pull official CentOS 8 support when RedHat decided that it is a good idea to break the repository the minute official support ends. It broke our CI setup. Without regular CI testing, we can´t guarantee that the installation instructions work. CentOS has always been rather maintenance-intensive and the unclear state between the stream and stable version has not improved the situation. I simply don't have the capacity to maintain official instructions. If somebody wants to set up and maintain community instructions for RockyLinux here in the discussion section, I'd be happy to link the post from the official documentation.
